src/utils/UpdateNotification.js

```diff
@@ -372,6 +372,21 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     *
+     * @param {boolean} enable
+     */
+    function enableUpdateNotificationIcon(enable) {
+        var $updateNotification = $("#update-notification");
+        _addedClickHandler = !enable;
+
+        if(enable) {
+            $updateNotification.removeClass("update-in-progress");
+        } else {
+            $updateNotification.addClass("update-in-progress");
+        }
```

Comment:
Maybe you can use `$updateNotification.toggleClass("update-in-progress", !enable)` instead of the if block.